src/components/tree/tree.tsx

```diff
@@ -283,7 +283,7 @@ export class Tree {
       }
 
       // When focus is on a child node that is also either an end node or a closed node, moves focus to its parent node.
-      const parentItem = target.parentElement.closest("calcite-tree-item");
+      const parentItem = target.parentElement?.closest("calcite-tree-item");
```

Comment:
Per our convo, this and the updates below (`updateAncestorTree`) wouldn't need the optional chaining operator as the event is processed after it bubbles, so we can assume there's an ancestor chain. This can be refactored later.